### PR TITLE
Removed a lot of lint warnings

### PR DESCRIPTION
**Pull request by ADK0010**
#859

### CODE REVIEW

File: src/passes/inlineAssemblyTransformer.ts [hunk 1]
```diff
@@ -217,22 +216,22 @@ export class InlineAssemblyTransformer extends ASTMapper {
...
     });
-    const block: Block = new UncheckedBlock(ast.reserveId(), this.yul(node).src, statements);
+    const block: Block = new UncheckedBlock(ast.reserveId(), this.yul({ node }).src, statements);
     ast.replaceNode(node, block);
   }
 
   visitInlineAssembly(node: InlineAssembly, ast: AST): void {
-    this.verify(node, ast);
-    this.transform(node, ast);
+    this.verify({ node, ast });
```
Comment (swapnilraj) [inline src/passes/inlineAssemblyTransformer.ts:233]: Why are wrapping the arguments in an object?
What lint warning is it solving, is there a way we can modify the lint rule instead

**Comment by rjnrohit** [comment]
I would suggest running the command: `yarn tsc && yarn lint --fix` on your local and fix it before if you'd get some errors/warnings

**Comment by swapnilraj** [comment]
Hey @ADK0010 are you still working on this?

**Comment by ADK0010** [comment]
No

**Comment by rjnrohit** [comment]
Closing it, as author is no longer working on this.

**Pull request closed without merging**